src/plugins/chart_expressions/expression_heatmap/public/components/heatmap_component.tsx

```diff
@@ -287,15 +287,21 @@ export const HeatmapComponent: FC<HeatmapRenderProps> = memo(
 
     const bands = ranges.map((start, index, array) => {
       // by default the last range is right-open
-      let end = index === array.length - 1 ? Infinity : array[index + 1];
+      let end = index === array.length - 1 ? Number.POSITIVE_INFINITY : array[index + 1];
       // if the lastRangeIsRightOpen is set to false, we need to set the last range to the max value
       if (args.lastRangeIsRightOpen === false) {
-        const lastBand = max === start ? Infinity : endValue;
+        const lastBand = max === start ? Number.POSITIVE_INFINITY : endValue;
         end = index === array.length - 1 ? lastBand : array[index + 1];
       }
-      const overwriteArrayIdx = `${metricFormatter.convert(start)} - ${metricFormatter.convert(
-        end
-      )}`;
+
+      let overwriteArrayIdx;
+
+      if (end === Number.POSITIVE_INFINITY) {
+        overwriteArrayIdx = `≥ ${start}`;
```

Comment:
@stratoula not sure that it's the best fix, but I see from chart we return label in that format